src/main/resources/samples/mentee_template.json

```
@@ -0,0 +1,303 @@
+              "Machine Learning",
+              "Engineering management",
+              "Network Engineering",
+              "Security",
+              "Site Reliability Engineering"
+            ]
+          },
+          {
+            "2": [
+              "Quality Assurance",
+              "Backend Developer",
+              "Frontend Developer",
+              "Fullstack Developer",
+              "DevOps",
+              "Distributed Systems",
+              "Data Science",
+              "Data Engineering",
+              "Android",
+              "iOS",
+              "Business Analysis",
+              "Product Management",
+              "Project Management",
+              "Machine Learning",
+              "Engineering management",
+              "Network Engineering",
+              "Security",
+              "Site Reliability Engineering"
+            ]
+          },
+          {
+            "3": [
+              "Quality Assurance",
+              "Backend Developer",
+              "Frontend Developer",
+              "Fullstack Developer",
+              "DevOps",
+              "Distributed Systems",
+              "Data Science",
+              "Data Engineering",
+              "Android",
+              "iOS",
+              "Business Analysis",
+              "Product Management",
+              "Project Management",
+              "Machine Learning",
+              "Engineering management",
+              "Network Engineering",
+              "Security",
+              "Site Reliability Engineering"
+            ]
+          },
+          {
+            "4": [
+              "Quality Assurance",
+              "Backend Developer",
+              "Frontend Developer",
+              "Fullstack Developer",
+              "DevOps",
+              "Distributed Systems",
+              "Data Science",
+              "Data Engineering",
+              "Android",
+              "iOS",
+              "Business Analysis",
+              "Product Management",
+              "Project Management",
+              "Machine Learning",
+              "Engineering management",
+              "Network Engineering",
+              "Security",
+              "Site Reliability Engineering"
+            ]
+          },
+          {
+            "5": [
+              "Quality Assurance",
+              "Backend Developer",
+              "Frontend Developer",
+              "Fullstack Developer",
+              "DevOps",
+              "Distributed Systems",
+              "Data Science",
+              "Data Engineering",
+              "Android",
+              "iOS",
+              "Business Analysis",
+              "Product Management",
+              "Project Management",
+              "Machine Learning",
+              "Engineering management",
+              "Network Engineering",
+              "Security",
+              "Site Reliability Engineering"
+            ]
+          }
+        ],
+        "languages": [
```

languages the same as areas: 

```suggestion
        "languages": [   "Java", "C", "C++"] 
```